src/Compilers/Server/VBCSCompiler/App.config

```diff
@@ -7,7 +7,6 @@
   </startup>
   <runtime>
     <gcServer enabled="true"/>
-    <gcConcurrent enabled="false"/>
```

Comment:
Concurrent GC could be used to sacrifice some throughput and gain interactivity. Since we have a server process whose primary perf metric is throughput is that desirable? 
Were there any measurements done that this actually improves E2E compile times?

Reply:
It is possible that GC could pick some spare cycles when compiler does not saturate the CPU and get some advantage from that, but if that does not happen often enough, the added complexity and context switches may actually reduce the overall throughput.

I think we should measure before making this change.

Reply:
Agree. Need to measure here.